src/preparse.rs

```diff
@@ -90,6 +91,20 @@ impl<F> PreParsedSubtables<'_, F> {
             .find_map(|t| t.glyph_index(c.into()))
     }
 
+    /// Maps a variation of a character to a `GlyphId` using pre-parsed unicode cmap subtables.
+    #[inline]
+    pub fn glyph_variation_index(&self, c: char, v: char) -> Option<GlyphId> {
+        match self
+            .subtables
+            .cmap
+            .iter()
+            .find_map(|t| t.glyph_variation_index(c.into(), v.into()))
+        {
+            Some(GlyphVariationResult::Found(id)) => Some(id),
+            _ => None,
+        }
```

Comment:
This would better match the `Face` version I think
```suggestion
        self.subtables
            .cmap
            .iter()
            .find_map(|t| t.glyph_variation_index(c.into(), v.into()))
            .and_then(|r| match r {
                cmap::GlyphVariationResult::Found(v) => Some(v),
                cmap::GlyphVariationResult::UseDefault => self.glyph_index(c),
            })
```